client/src/lifecycleManager.js

```diff
@@ -559,5 +559,19 @@ class LifecycleManager extends LuigiClientBase {
       anchor
     });
   }
+
+  /**
+   * This function allows you to change node lables within the same view group, e.g. in your node config: `label: 'my Node {viewGroupData.vg1}'`.
+   * @since NEXTRELEASE
+   * @param {Object} data
```

Comment:
It would be good to add some small description of the parameter.
(To apply the change, please run `npm install` or `lerna run docu` as mentioned [here](https://github.com/SAP/luigi/blob/main/docs/content-guidelines.md#API-documentation), because the API .md files can't be edited directly, they are generated from the .js files.)

```suggestion
   * @param {Object} data a data object containing the view group name and desired label
```